### PR TITLE
Fix redirect bug on page loads without lang

### PR DESCRIPTION
**Pull request by corwintines**
## Description
When going to a page without a lang the site gets in a redirect loop and never loads. Added back in redirects removed in https://github.com/ethereum/ethereum-org-website/pull/5097/files to resolve

### CODE REVIEW

**Comment by wackerow** [comment]
Getting the same homepage bug on Brave mobile (iPhone) with the preview link for this PR
https://ethereumorgwebsitedev01-redirectissue.gtsb.io/

Suggests to me there may be a deeper problem than just these redirect

File: src/components/Layout.js [hunk 1]
```diff
@@ -76,26 +76,26 @@ const Layout = (props) => {
...
-    }
-
-    if (props.location.hash && !props.location.hash.includes("gatsby")) {
-      const idTag = props.location.hash.split("#")
-      if (document.getElementById(idTag[1]) !== null) {
-        document.getElementById(idTag[1]).scrollIntoView(false)
-      }
-    }
-  }, [props.path, props.location])
+  // useEffect(() => {
```
Comment (samajammin) [inline src/components/Layout.js:79]: Why comment all this out?

**Review by samajammin — CHANGES_REQUESTED** [review]
Redirects look good but let's undo that commented code.

**Comment by wackerow** [comment]
So I'm still getting the redirect when I try this on mobile, both Brave and Safari with private tabs. This is a Gatsby link though, and our main concern is fixing prod, which is Netlify, so I'll bring this in to see if it fixes it on Netlify side.

**Pull request merged**